bower.json

```diff
@@ -17,7 +17,7 @@
     "bootstrap": ">=3.1.0 <4",
     "moment": ">=2.6.0",
     "jquery-extendext": ">=0.1.2",
-    "doT": ">=1.0.3"
+    "doT": ">=1.0.3",
```

Comment:
the JSON is now invalid (check TravisCI)

Reply:
yes, a trailing comma was there, fixed it